docs/zh/latest/how-to-build.md

```diff
@@ -113,9 +113,22 @@ sudo yum install ./apisix/*.rpm
   make install
   ```
 
-- 4.1 `make deps` 安装 `lualdap` 失败, 错误信息如: `Could not find header file for LDAP`
-
-      解决方案: 通过 `luarocks config` 手动设置 `LDAP_DIR` 变量, 比如 `luarocks config variables.LDAP_DIR /usr/local/opt/openldap/`
+   **注意**：如果使用 `make deps` 安装 `lualdap` 失败, 错误信息如： `Could not find header file for LDAP`，那么可以肯定这是一类关于安装依赖失败的问题。同理，遇到 `PCRE`、`openssl` 等依赖包安装失败问题，皆可以使用本方法解决问题。
+
+  解决思路：根据 luarocks 支持添加自定义包到项目中的思想，使用第三方工具安装缺失的包，并将其地址添加到项目的配置文件中以识别。
```

Comment:
```suggestion
  解决思路：luarocks 支持自定义编译时依赖目录，使用第三方工具安装缺失的依赖，并将其文件路径添加到 luarocks 的变量表中。
```

Reply:
what about add link? https://github.com/luarocks/luarocks/wiki/Config-file-format#variables

Reply:
I think that's a good idea.

Reply:
hi @Chever-John , PLAT suggested changes above